Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -1,41 +1,137 @@
+                  <Dropdown.Item icon={HiOutlineCog}>
+                    Settings
+                  </Dropdown.Item>
+                  <Dropdown.Divider />
+                  <Dropdown.Item icon={HiOutlineLogout} onClick={handleLogout}>
+                    Sign out
+                  </Dropdown.Item>
+                </Dropdown>
+              </>
+            ) : (
+              <button 
+                onClick={() => window.location.href = '/login'}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)